src/org/zaproxy/zap/extension/alertFilters/ContextAlertFilterPanel.java

```diff
@@ -103,11 +103,9 @@ public AlertFiltersMultipleOptionsPanel(ExtensionAlertFilters extension, AlertFi
 			super(model);
 			this.extension = extension;
 
-			getTable().getColumnExt(0).setPreferredWidth(20);
-			getTable().getColumnExt(1).setPreferredWidth(50);
-			getTable().getColumnExt(2).setMaxWidth(30);
-			getTable().getColumnExt(3).setPreferredWidth(20);
+			getTable().getColumnExt(0).setMaxWidth(70);
```

Comment:
Is the maximum really needed? With some languages it cuts the word, for example, "Ενεργοποιημένο" is shown as "Ενεργο..." and "Ingeschakeld" as "Ingesc...".